packages/jest-console/src/getConsoleOutput.ts

```diff
@@ -5,43 +5,73 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import * as path from 'path';
 import chalk = require('chalk');
-import slash = require('slash');
+import {
+  StackTraceConfig,
+  StackTraceOptions,
+  formatStackTrace,
+} from 'jest-message-util';
 import type {ConsoleBuffer} from './types';
 
 export default (
   root: string,
   verbose: boolean,
   buffer: ConsoleBuffer,
+  optionalStackTraceConfig?: StackTraceConfig,
```

Comment:
I wonder if we should get passed `Config.ProjectConfig` here rather than `StackTraceConfig`

Reply:
Sounds like a good idea, currently, all the places calling `getConsoleOutput()` have access to project config; and if `getConsoleOutput()` needs other entries in project config in the future, no code change needed. 

I think I will making this parameter non-optional, and make sure all existing callers do pass project config :)